go/libraries/doltcore/sqle/dtables/diff_table.go

```diff
@@ -685,30 +685,39 @@ func (dp DiffPartition) GetRowIter(ctx *sql.Context, ddb *doltdb.DoltDB, joiner
 // isDiffablePartition checks if the commit pair for this partition is "diffable".
 // If the primary key sets changed between the two commits, it may not be
 // possible to diff them.
```

Comment:
Probably helpful for future code readers to explain `simpleDiff` and `fuzzyDiff` in the function docs and maybe how they are intended to be used.